### PR TITLE
Change docs usage of aws.ami to amazon.ami to be consistent with terraform

### PR DESCRIPTION
**Pull request by cbednarski**

### CODE REVIEW

File: website/source/docs/post-processors/atlas.html.markdown [hunk 1]
```diff
@@ -37,7 +37,7 @@ The configuration allows you to specify and access the artifact in Atlas.
   have access to the organization, hashicorp in this example, in order to add an artifact to
   the organization in Atlas.
 
-* `artifact_type` (string) - For uploading AMIs to Atlas, `artifact_type` will always be `aws.ami`.
+* `artifact_type` (string) - For uploading AMIs to Atlas, `artifact_type` will always be `amazon.ami`.
   This field must be defined because Atlas can host other artifact types, such as Vagrant boxes.
 
 -> **Note:** If you want to upload Vagrant boxes to Atlas, for now use the [Vagrant Cloud post-processor](/docs/post-processors/vagrant-cloud.html).
```
Comment (pearkes) [inline website/source/docs/post-processors/atlas.html.markdown]: Oh gosh, this should be removed too.

**Comment by pearkes** [comment]
:+1:

**Comment by mitchellh** [comment]
LGTM

File: website/source/docs/post-processors/atlas.html.markdown (hunk 1 above)
Comment (cbednarski) [inline website/source/docs/post-processors/atlas.html.markdown]: Oops, just caught this.

**Pull request merged**